test/remote/gateways/remote_kushki_test.rb

```diff
@@ -51,6 +51,50 @@ def test_failed_purchase
     assert_equal 'Monto de la transacción es diferente al monto de la venta inicial', response.message
   end
 
+  def test_successful_authorize
+    # Kushki only allows preauthorization for PEN, CLP, and UF.
```

Comment:
This doesn't seem to actually be true in practice; the `test_successful_capture` runs with the default USD currency and everything goes fine. 🤷‍♀